packages/runtime/src/widgets/SideBar.tsx

```diff
@@ -35,7 +35,7 @@ interface MenuItem {
 }
 
 interface MenuBaseProps {
-  items: MenuItem[];
+  menuItems: MenuItem[];
```

Comment:
Is this because of the parser changes?

Reply:
yes, since menu items doesn't contain any widgets, it breaks.

Reply:
Can you check if this is still an issue after rebasing? I separated menu out from widgets and it's on another parsing path now.

If it's still an issue, let's just move the widget unwrapping into the TabBar component instead of the parser level.

Reply:
not an issue anymore with menu moved out from the widgets